todo-vue/src/components/Home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="home">
+    <header><h1>Todo App</h1></header>
+    <main>
+      <ul>
+        <CardComponent
+          v-for="todo in todos"
+          :key="todo.id"
+          :title="todo.title"
+          :id="todo.id"
+          @setTitle="editTodo"
+          @delete="deleteTodo"
+        />
+      </ul>
+      <form @submit="addTodo">
+        <input id="new-todo-input" type="text" placeholder="add new todo..." />
+      </form>
```

Comment:
```suggestion
      <form @submit.prevent="addTodo">
        <input id="new-todo-input" type="text" placeholder="add new todo..." />
      </form>
```
이렇게 아예 이벤트를 전달할 때 prevent를 할 수 있습니다 ㅎㅎ
비슷하게 
`@submit.stop`일 하면 이벤트 전파를 방지할 수도 있어요!